src/graphql/schema.test.js

```diff
@@ -152,6 +152,24 @@ describe("graphql/schema", () => {
         'field "O"/"foo" has invalid type "Color" of kind "ENUM"'
       );
     });
+    it("disallows objects with node fields of unfaithful type", () => {
```

Comment:
This test name is wrong. Node fields of unfaithful type are and should
be allowed. What’s disallowed is unfaithful fidelities whose lists of
actual typenames include a type that does not exist.